go-controller/pkg/cni/ovs.go

```diff
@@ -112,6 +112,16 @@ func waitForPodFlows(mac string, ifAddrs []*net.IPNet) error {
 		// TODO(trozet): in the future use a more stable mechanism provided by OVN:
 		// https://bugzilla.redhat.com/show_bug.cgi?id=1839102
 
+		// ensure the OVS interface still exists. It may have been removed by a subsequent CNI ADD
+		// and if the interface doesn't exist, there's no need to keep checking for flows
+		out, err := ovsFind("Interface", "_uuid", "name="+ifName)
```

Comment:
@trozet I don't think this is quite right. It's not that the whole Interface record is removed, it's that we remove the iface-id external id from the interface record:

`if out, err := ovsExec("remove", "Interface", uuid, "external-ids", "iface-id"); err != nil {`

so you'd want to look for name=ifName *and* external-ids:iface-id=fmt.Sprintf("%s_%s", namespace, podName)

Reply:
oops my bad. will fix